.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+ - "3.6.1"
+# Install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install coveralls
+# Run tests
+script:
+  nosetests --with-coverage --cover-package=tests && coverage report
+after_success:
+  - coveralls
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database shopping_list_api_test;' -U postgres
+addons:
+  postgresql: "9.6"
```

Comment:
Add a new line after line 18. Also, do this for the other files that don't have a blank line at the end of the file.